<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCategory</class>
 <widget class="QWidget" name="ImportCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Category</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
     <item row="1" column="0">
      <widget class="QLabel" name="u_catPathLabel">
       <property name="text">
        <string>Category Path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="u_nameEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="u_nameLabel">
       <property name="text">
        <string>Template Name</string>
       </property>
       <property name="buddy">
        <cstring>u_nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="u_addCatButton">
       <property name="text">
        <string>Add category</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="u_categoryEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="u_newCatLabel">
       <property name="text">
        <string>New Category</string>
       </property>
       <property name="buddy">
        <cstring>u_categoryEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="u_addSubCatButton">
       <property name="text">
        <string>Add sub-category</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QTreeView" name="u_catTreeView">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLabel" name="u_categoryLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="u_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>u_catTreeView</tabstop>
  <tabstop>u_nameEdit</tabstop>
  <tabstop>u_categoryEdit</tabstop>
  <tabstop>u_addCatButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
